examples/example-server-lib/decoration_provider.cpp

```diff
@@ -38,6 +38,8 @@
 #include <system_error>
 #include <iostream>
 
+namespace geom = mir::geometry;
```

Comment:
My current preference is for `using namespace mir::geometry`, but that's non-blocking.